src/schema/filter_artworks.js

```diff
@@ -286,6 +287,8 @@ function filterArtworks(primaryKey) {
         delete gravityOptions.medium
       }
 
+      removeNulls(gravityOptions)
```

Comment:
This came up since we have some params that we want to be able to set (filters such as gallery/institution, time period, etc.), but which _dont_ have meaningful defaults. Relay will always send the same set of params, which winds up sending `null` for those (which Gravity can't handle).

That (and another historical reason) is why we _do_ allow defaults (which just get dropped), such as `*` and `*-*` for values and ranges (a few lines up we do this for `medium`).

Reply:
Ah, this answers my question above 👍